hera_cal/smooth_cal.py

```diff
@@ -257,26 +301,40 @@ def rephase_to_refant(gains, refant, flags=None):
 
 class CalibrationSmoother():
 
-    def __init__(self, calfits_list, flag_file_list=[], flag_filetype='h5', pick_refant=False, antflag_thresh=0.0, verbose=False):
+    def __init__(self, calfits_list, flag_file_list=[], flag_filetype='h5', antflag_thresh=0.0,
+                 pick_refant=False, freq_threshold=1.0, time_threshold=1.0, ant_threshold=1.0, verbose=False):
```

Comment:
I don't _really_ care either way, but it seems odd to have the top level wrapper have a set of `freq_threshold` and `time_threshold` defaults that are different from this, but the same as the function that actually uses these keywords.

Reply:
Yeah, I thought about that a bunch. I'm a little wary of making the flagging automatic (changing the default of the constructor), in part because CalibrationSmoother is a really useful way to load a bunch of calibration solutions and make them into a big grid (it's way, way faster than loading a list with pyuvdata). But I figured if you're going to explicitly call the more general function, you'll want to use the default parameters. Also, if you're calling it from a script. 

So yeah, it's a bit odd, but I tried to make it clear in the docstrings.

Reply:
I figured you had a thought process like that. I'm fine with this solution, but just to throw another idea out there: you could have the default thresholds set to the 0.35 and 0.5 like the function, but have another keyword that determines whether to actually call that function or not. That keyword would default to False here.

Reply:
I thought about that too... I find extra keywords that control whether keywords get used to be confusing and try to avoid them. It's too easy to change the settings but not flip the switch and be surprised when nothing happens.